### PR TITLE
Create uk_ua.json

### LINKED ISSUE #520: Add Ukraine language ( uk-UA )
[uk-UA.txt](https://github.com/Wurst-Imperium/Wurst7/files/7833799/uk-UA.txt)
The translation was done by xVirum, a good friend of mine who lives in Hohland (Ucraine)
I fixed what was said in #526

### PR DESCRIPTION
**Pull request by KTibow**
Co-authored-by: xVirum <xVirum@users.noreply.github.com>
Co-authored-by: MrYotic <MrYotic@users.noreply.github.com>

## Description
Adds a Ukraine translation. Requested by MrYotic, translated by xVirum, fixed into valid JSON by me. Closes #520.

## (Optional) screenshots / videos
Not tested since I don't know how to build Wurst.

### CODE REVIEW

**Review by Alexander01998 — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/wurst/lang/uk_ua.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  "description.wurst.hack.boatfly": "Дозволяє літати у човні.",
+  "description.wurst.hack.bonemealaura": "Автоматично використовує кісткове борошно на рослинах.\nПозначте меню, на яких типах рослин використовувати.",
+  "description.wurst.hack.bowaimbot": "Автоматично цілиться із луку або арбалета.",
+  "description.wurst.hack.buildrandom": "Ставить навколо блоків у випадковому порядку.",
+  "description.wurst.hack.bunnyhop": "Примушує вас постійно стрибати.",
+  "description.wurst.hack.cameranoclip": "Дозволяє камеру від 3-ї особи пройти через стіни.",
+  "description.wurst.hack.cavefinder": "Допомагає знайти печери, підсвічуючи їх вибраним кольором.",
+  "description.wurst.hack.chattranslator": "Всі вхідні повідомлення в чаті будуть перекладатися Google перекладачем.",
+  "description.wurst.hack.chestesp": "Підсвічує вибраним кольором скрині поблизу.",
+  "description.wurst.hack.clickaura": "Автоматично атакує найближчу сутність, коли ви натискаєте ЛКМ.\n\n§c§lУВАГА:§r ClickAura'и звичайні виглядають більш підозріло, ніж Killaura'и і легше виявляються плагінами. використовувати Killaura або TriggerBot.",
```
[the remark this answers is not in the document]
Comment (xVirum) [reply src/main/resources/assets/wurst/lang/uk_ua.json:42]: \n\n§c§lУВАГА:§r ClickAura'и звичайні виглядають більш підозріло, ніж Killaura'и і легше виявляються плагінами. використовувати Killaura або TriggerBot. 
This is in the text now
text

[the remark this answers is not in the document]
Comment (xVirum) [reply src/main/resources/assets/wurst/lang/uk_ua.json:42]: There is a warning in the text, but its translation is inaccurate, I will correct it soon.

**Comment by MrYotic** [comment]
[uk-UA.txt](https://github.com/Wurst-Imperium/Wurst7/files/7833787/uk-UA.txt)
I fixed that

File: src/main/resources/assets/wurst/lang/uk_ua.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (xVirum) [reply src/main/resources/assets/wurst/lang/uk_ua.json:42]: fixed in @MrYotic txt file

[the remark this answers is not in the document]
Comment (KTibow) [reply src/main/resources/assets/wurst/lang/uk_ua.json:42]:
```suggestion
  "description.wurst.hack.clickaura": "Автоматично атакує найближчу сутність, коли ви натискаєте ЛКМ.\n\n§c§lУВАГА:§r ClickAura'и звичайно виглядають більш підозріло, ніж Killaura'и і легше виявляються плагінами. Краще використовувати Killaura або TriggerBot.",
```

**Comment by xVirum** [comment]
fixed.

**Review by Alexander01998 — APPROVED** [review]

**Pull request merged**